src/System.Net.Requests/src/System/Net/HttpDateParse.cs

```diff
@@ -0,0 +1,465 @@
+//------------------------------------------------------------------------------
+// <copyright file="_HTTPDateParse.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System.Globalization;
+
+namespace System.Net {
```

Comment:
bracing style needs to be "Allman" style.  Pls see CoreFx coding guidelines.
But basically braces on their own lines.